hr_webcam/hr_employee.py

```diff
@@ -0,0 +1,22 @@
+from openerp import models
+
+
+class hr_employee(models.Model):
+    _name = 'hr.employee'
```

Comment:
Here you wanted to remove `_name` and not `_inherit` see https://github.com/yoyo2k/hr/commit/05296ab2269cf4397a5ef25bb3c0dd0074da28a2